contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -2540,6 +2540,146 @@ CREATE OR REPLACE FUNCTION sys.json_query(json_string text, path text default '$
 RETURNS sys.NVARCHAR
 AS 'babelfishpg_tsql', 'tsql_json_query' LANGUAGE C IMMUTABLE PARALLEL SAFE;
 
+--JSON_MODIFY
+/*
+This function is used to update the value of a property in a JSON string and returns the updated JSON string.
+
+It has been implemented in three parts:
+1) Set the append and create_if_missing flag as postgres functions do not directly take append and lax/strict mode in the jsonb_path.
+2) To convert the input path into the expected jsonb_path.
+3) To implement the main logic of the JSON_MODIFY function by dividing it into 8 different cases.
+
+*/
+CREATE OR REPLACE FUNCTION sys.json_modify(in expression sys.NVARCHAR,in path_json TEXT, in new_value TEXT)
```

Comment:
Are input (nvarchar, text, text) and output (nvarchar) data types consistent with T-SQL's?